tests/integration/test_s3_zero_copy_ttl/test.py

```diff
@@ -1,94 +0,0 @@
-#!/usr/bin/env python3
-import time
-
-import pytest
-from helpers.cluster import ClickHouseCluster
-
-cluster = ClickHouseCluster(__file__)
-node1 = cluster.add_instance(
-    "node1", main_configs=["configs/s3.xml"], with_minio=True, with_zookeeper=True
-)
-node2 = cluster.add_instance(
-    "node2", main_configs=["configs/s3.xml"], with_minio=True, with_zookeeper=True
-)
-node3 = cluster.add_instance(
-    "node3", main_configs=["configs/s3.xml"], with_minio=True, with_zookeeper=True
-)
-
-
-@pytest.fixture(scope="module")
-def started_cluster():
-    try:
-        cluster.start()
-
-        yield cluster
-    finally:
-        cluster.shutdown()
-
-
-def test_ttl_move_and_s3(started_cluster):
```

Comment:
This test looks mostly fine too, failures in https://github.com/ClickHouse/ClickHouse/pull/46946, https://github.com/ClickHouse/ClickHouse/pull/49631 and https://github.com/ClickHouse/ClickHouse/pull/51135 probably were for a reason: [CI DB](https://play.clickhouse.com/play?user=play#d2l0aCAKJyV0ZXN0X3R0bF9tb3ZlX2FuZF9zMyUnIGFzIG5hbWVfcGF0dGVybiwKJyUlJyBhcyBjb250ZXh0X3BhdHRlcm4sCicyMDIzLTAxLTAxJyBhcyBzdGFydF9kYXRlLAooNDU0NjEsIDQ1MjM2LCA0NDk4NSwgNDc0NzUpIGFzIG5vaXN5X3BycywKKCdTdGF0ZWxlc3MgdGVzdHMgKGFzYW4pJywgJ1N0YXRlbGVzcyB0ZXN0cyAoYWRkcmVzcyknLCAnU3RhdGVsZXNzIHRlc3RzIChhZGRyZXNzLCBhY3Rpb25zKScpIGFzIGJhY2twb3J0X2FuZF9yZWxlYXNlX3NwZWNpZmljX2NoZWNrcwpzZWxlY3QgCnRvU3RhcnRPZkRheShjaGVja19zdGFydF90aW1lKSBhcyBkLApjb3VudCgpLCAgZ3JvdXBVbmlxQXJyYXkocHVsbF9yZXF1ZXN0X251bWJlciksIGFueShyZXBvcnRfdXJsKQpmcm9tIGNoZWNrcyB3aGVyZSBzdGFydF9kYXRlIDw9IGNoZWNrX3N0YXJ0X3RpbWUgYW5kIHB1bGxfcmVxdWVzdF9udW1iZXIgbm90IGluIAooc2VsZWN0IHB1bGxfcmVxdWVzdF9udW1iZXIgYXMgcHJuIGZyb20gY2hlY2tzIHdoZXJlIHBybiE9MCBhbmQgc3RhcnRfZGF0ZSA8PSBjaGVja19zdGFydF90aW1lIGFuZCBjaGVja19uYW1lIGluIGJhY2twb3J0X2FuZF9yZWxlYXNlX3NwZWNpZmljX2NoZWNrcykgCmFuZCB0ZXN0X25hbWUgaWxpa2UgbmFtZV9wYXR0ZXJuIGFuZCAodGVzdF9jb250ZXh0X3JhdyBpbGlrZSBjb250ZXh0X3BhdHRlcm4pCmFuZCBwdWxsX3JlcXVlc3RfbnVtYmVyIG5vdCBpbiBub2lzeV9wcnMgCmFuZCB0ZXN0X3N0YXR1cyBpbiAoJ0ZBSUwnLCAnRkxBS1knLCAnZmFpbHVyZScpIGdyb3VwIGJ5IGQgb3JkZXIgYnkgZCBkZXNj)